packages/app/jest.config.js

```diff
@@ -9,10 +9,9 @@ const DEPS_TO_TRANSFORM = [
   'd3-time',
   'd3-time-format',
   'internmap',
-  'three',
```

Comment:
The ESM entrypoint is now properly referenced in Three's `package.json`, so this is no longer necessary.